streams/blocks.py

```diff
@@ -0,0 +1,13 @@
+from wagtail.core import blocks
+
+class TitleBlock(blocks.StructBlock):
+    text = blocks.CharBlock(
+        requried=True,
+        help_text='Text to display',
+    )
+
+    class Meta:
+        template = "streams/title_block.html"
+        icon = " edit"
+        label = "Title"
+        help_text = " Center text to display"
```

Comment:
Careful with spaces at the start of a string. " edit" vs "edit" may not be the same thing in all cases.